### ISSUE TITLE
Accessibility/USWDS audit for compliance

### ISSUE THREAD

**Issue by Sgtpluck** [18F/tock#1542]
**Point of contact on this issue**
Who can we follow-up with if we have questions?
@kfoley-18F 

**Billable?**
- [ ] Yes
- [x] No
tock code:
1109

**Skills needed**

- [ ] Any human
- [x] Design
- [ ] Content
- [x] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [x] No

How much time do you anticipate this work taking?
1-2 weeks

### A description of the work

Tock needs to be in accessibility compliance. Do an accessibility audit to see where Tock does not meet compliance standards and create a follow-up ticket to implement changes.

This may require adding pa11y or equivalent tooling to Tock's build pipeline.

### Acceptance Criteria

What does "done" look like?

A list of changes that need to be made in a ticket.

**Comment by sarahmhgov** [comment]
@kfoley-18F would a reasonable first step here be using the 18F accessibility checklist to write an audit?

new 18f-er, familiar w 2.0 AA compliance but want to make sure aligned on approach.

**Comment by kfoley-18F** [comment]
@sarahmhgov - yes! and then open some tickets on the Tock repo (which will be copied into the DA team repo via automation) on things to fix in future iterations.

**Comment by sarahmhgov** [comment]
Update from Mar 3- 

- accessibility tickets have been created in tock
- summary here: https://docs.google.com/presentation/d/1hyvTanixtUDeLZEAwvXKiSbaldSMunTNS2sjBFJD7eo/edit?usp=sharing

**Comment by sarahmhgov** [comment]
Adding CI tooling was not scope for this work, suggest creating a second ticket if that is a priority.

**Comment by colinmurphy01** [comment]
@sarahmhgov who might be the best person to review this?

**Comment by jasnakai** [comment]
These all look good. They are all definitely accessibility issues.
NOTE: This is not a complete list of accessibility issues. A more thorough audit is needed.
